src/cmr/concepts/concept.js

```diff
@@ -205,6 +198,128 @@ export default class Concept {
     return 0
   }
 
+  /**
+   * Fetches missing items for a umm or json endpoint with retry logic
+   * @param {Array} missingIds - Array of concept IDs for missing items
+   * @param {Array} keys - Array of requested keys to fetch for each item (ummKeys or jsonKeys)
+   * @param {Function} fetchFunction - Function to fetch data from CMR (fetchUmm or fetchJson)
+   * @param {Function} parseFunction - Function to parse the fetched data
+   * @param {Integer} retryCount - Current retry attempt count
+   */
+  async fetchWithRetry(missingIds, keys, fetchFunction, parseFunction, retryCount = 0) {
+    const {
+      maxRetries,
```

Comment:
These values are strings coming from env variables. Be sure to update the mock values in your tests as well

Reply:
Updated the mock